styles/common.css

```diff
@@ -1,17 +1,71 @@
+html, body {
+    margin: 0;
+    padding: 0;
+}
+
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    background-color: #DCA47C;
+}
+
+
 .wrapper {
     display: flex;
     width: 100%;
-    height: 100vh;
-    background: lavender;
+    background: #FCF8F3;
+    justify-content: center;
+}
+
+
+.content {
+    max-width: fit-content;
+}
+
+
+.page-header {
+    display: flex;
     justify-content: space-between;
+    align-items: center;
 }
 
-.sidebar {
-    width: 30%;
-    background: beige;
+
+h1 {
+    font-size: 30pt;
+    font-family: "Roboto Slab", serif;
+    font-optical-sizing: auto;
+    font-weight: 600;
+    font-style: normal;
 }
 
-.content {
-    width: 60%;
-    background: beige;
-}
\ No newline at end of file
+/* Button styling */
+button {
+    border: none;
+    background-color: transparent;
+}
+
+.home-button {
+    align-content: center;
+    margin-left: 30px;
+}
+
+div.menu-items > a > button {
+    font-family: "Roboto Slab", serif;
+    font-optical-sizing: auto;
+    font-weight: 400;
+    font-style: normal;
+    margin: 20px 15px;
+    width: 100px;
+    height: 50px;
+    background-color: #DCA47C;
+    border: none;
+    border-radius: 6.5px;
+    font-size: larger;
+}
+
+/* Active page button highlighting */
+div.menu-items > a > .page-button {
```

Comment:
if it active, mention it in the class name.
for instance,, you "menu-items-button" call this one "menu-items-button-active"

Reply:
Please read about BEM.

Reply:
The active page button now has an additional class of navbar-button-active.